### PR TITLE
Call the serializer while running access policy workflows

### LINKED ISSUE #1229: Input data for namespaces are not sanitized
```
curl -X POST -H 'Content-Type: application/json' -H 'Authorization: Basic YWRtaW46cGFzc3dvcmQ=' -d '[]' http://localhost:5001/pulp/api/v3/pulp_container/namespaces/
```

```
pulp [62426a7547384c4a824fb23bf628621f]: django.request:ERROR: Internal Server Error: /pulp/api/v3/pulp_container/namespaces/
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/rest_framework/viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 497, in dispatch
    self.initial(request, *args, **kwargs)
  File "/src/pulpcore/pulpcore/app/viewsets/base.py", line 317, in initial
    super().initial(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 415, in initial
    self.check_permissions(request)
  File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 332, in check_permissions
    if not permission.has_permission(request, self):
  File "/usr/local/lib/python3.8/site-packages/rest_access_policy/access_policy.py", line 69, in has_permission
    allowed = self._evaluate_statements(statements, request, view, action)
  File "/usr/local/lib/python3.8/site-packages/rest_access_policy/access_policy.py", line 113, in _evaluate_statements
    matched = self._get_statements_matching_conditions(
  File "/usr/local/lib/python3.8/site-packages/rest_access_policy/access_policy.py", line 262, in _get_statements_matching_conditions
    passed = self._check_condition(condition, request, view, action)
  File "/usr/local/lib/python3.8/site-packages/rest_access_policy/access_policy.py", line 288, in _check_condition
    result = method(request, view, action)
  File "/src/pulp_container/pulp_container/app/global_access_conditions.py", line 91, in namespace_is_username
    namespace = request.data.get("name")
AttributeError: 'list' object has no attribute 'get'
```

### PR DESCRIPTION
**Pull request by lubosmj**
closes #1229

### CODE REVIEW

**Comment by mdellweg** [comment]
This is adding the overhead of serializing the input data twice, right? Too bad access policies is implemented not quite in the django way...

**Comment by lubosmj** [comment]
Yes, but now we catch the errors before it passes through the inner layers.

File: pulp_container/app/registry_api.py [hunk 1]
```diff
@@ -86,8 +84,6 @@
     SIGNATURE_TYPE,
 )
 
-FakeView = namedtuple("FakeView", ["action", "get_object"])
-
```
Comment (mdellweg) [inline pulp_container/app/registry_api.py:90]: So all uses of this had moved elsewhere before?

**Review by mdellweg — APPROVED** [review]
So the fake views are only in place for the token endpoint, right? Still not a fan of validating everything twice. But the current architecture seems to need it.

File: pulp_container/app/registry_api.py (hunk 1 above)
Comment (lubosmj) [reply pulp_container/app/registry_api.py:90]: It seems like it was introduced in https://github.com/pulp/pulp_container/commit/8ca2f0e88462f78c76423cce8af838d8d3676e85, but it was never ever used.

**Pull request merged**